src/tests/object/util.c

```diff
@@ -77,30 +74,30 @@ static int test_basic_check_for_inscrip_with_int(void *state) {
 
     /* An inscription not containing the search string should return zero and
      * leave inarg unchanged. */
-    obj.note = quark_add("@m1@b1@G1");
+    obj.note = string_make("@m1@b1@G1");
     inarg = dummy;
     eq(check_for_inscrip_with_int(&obj, "=g", &inarg), 0);
     eq(inarg, dummy);
 
```

Comment:
To avoid false positives for memory leaks if the test cases are run under a leak checker, I'd add `string_free(obj.note);` here and after the other checks below.